source/img/distro/pclinuxos.svg

```diff
@@ -0,0 +1,34 @@
+</svg>
```

Comment:
The current code in the svg file does not display the icon correctly. It seems that the layers are oversized, I suggest trying this: [PCLinuxOS.128px.svg.zip](https://github.com/flatpak/flatpak.github.io/files/5031305/PCLinuxOS.128px.svg.zip)

Reply:
@haggen88 feel free to make the changes you think are needed to this branch and open a new PR